src/input_utils.py

```diff
@@ -0,0 +1,87 @@
+import attr
+import tensorflow as tf
+import tensorflow_text as tf_text
+
+from typing import List
+
+from google_research.etcmodel.models.modeling import EtcConfig 
+
+
+@attr.s
+class PretrainInputConfig(object):
+  """Config options for pretraining model input."""
+
+  image_size = attr.ib(default=224)
+  text_keys = attr.ib(factory=List)
+  patch_size = attr.ib(default=16)
+  patch_order = attr.ib(default='snake')
+  mlm_use_whole_word = attr.ib()  # type: bool
```

Comment:
No default? Also, this one has a "# type" annotation and the other fields do not. Please unify.